docs/_Research-Data-Management/25-digital-preservation.md

```diff
@@ -5,40 +5,71 @@ layout: default
-3. **Step 3**: identify data that should be kept as it may have long-term value. 
-4. **Step 4**: weigh up the costs and identify any need for external advice in case of shortfall in the budget. 
-5. **Step 5**: complete the data appraisal, i.e. list what data must, should or could be kept to fulfil which potential reuse purposes and summarise any actions needed to prepare the data for deposit, or justification for not keeping it. 
-
-# Best practices
-Digitally preserving research data is usually considered the task of institutions and infrastructures. That being said, researchers can prepare their data in a way that facilitates digital preservation:
-* **Preparing data in a way that facilitates digital preservation**
-    * Documenting data with metadata and context information to ensure reusability over both the short- and long-term.
-    * Using well-known open formats with published specifications during the project phase.
-* **Following the 3-2-1 rule**
+Digital preservation means taking certain measures to ensure that digital material can be found and can be accessed in the long term ("long-term accessibility of data"). It aims to preserve information in a way that is understandable and reusable for a specific community and to prove its authenticity. 
+
+# Digital preservation for researchers
+The sustainable handling of data by researchers naturally facilitates the long-term accessibility of data. Best practice methods are:
+* Cleaning data / data structures - see also: [Data Organisation](https://knowledgebase.nfdi4microbiota.de/Research-Data-Management/14-data-organization.html)
+* Validating data - see also: [Data Quality Control](https://knowledgebase.nfdi4microbiota.de/Research-Data-Management/13-data-qc.html)
+* Documenting data with metadata and context information to ensure reusability: commenting, adding descriptive, administrative and technical metadata, asigning user license.
+* Using well-known open file formats during the project phase - see below - or transfering data into reusable file formats (needs documenting: original file or derivative)
+* Storing data following the 3-2-1 rule:
     * Keeping 3 copies of any important file
     * Storing files on 2 different media types
     * Keeping at least 1 copy off site.
 
-# Recommended preservation formats for research data
-For digital preservation, it is recommended saving files in the original software format and in an additional recommended file format, i.e. a format that is:
-* Open rather than proprietary
-* Exportable to / unpackable into an open format (e.g. xlsx, docx, etc. can be unpacked into folders of xml files)
-* Machine-readable
+## Data selection
+To decide well-founded on data selection we recommend reading the how-to guide ["Five steps to decide what data to keep"](https://www.dcc.ac.uk/guidance/how-guides/five-steps-decide-what-data-keep) by Angus Whyte. The suggested steps are:
```

Comment:
Thanks for this contribution! The author indicates how to cite his work:

> Please cite as: DCC (2014). 'Five steps to decide what data to keep: a checklist for appraising research data v.1'. Edinburgh: Digital Curation Centre. Available online: [/resources/how-guides](https://www.dcc.ac.uk/resources/how-guides)

Could you please edit the references.bib file to accommodate this reference?

In case a few resource can help:
- https://www.scribbr.com/apa-examples/website/
- or https://zbib.org/ to format a website reference into a bibtex

Reply:
Hi @UParmaksiz feel free to reach out if there is something unclear with my comments or if you need help somewhere. Other than this missing bibliography  entry and a few typos, this contribution is good to go ; )

Reply:
I did it in 7e0a18ff1a1a7dddbac6f54787df0dfc63099298